ops/jujuversion.py

```diff
@@ -38,6 +38,18 @@ def __eq__(self, other):
         return self.major == other.major and self.minor == other.minor\
             and self.tag == other.tag and self.build == other.build and self.patch == other.patch
 
+    def _less_than_property(self, other, prop):
+        self_prop = getattr(self, prop)
+        other_prop = getattr(other, prop)
+        if self_prop != other_prop:
```

Comment:
I think this reads a little better as:

```
if self_prop == other_prop:
  return None
if prop == 'tag':
  if not self_prop:
    return False
  if not other_prop:
    return True
return self_prop < other_prop
```

Reply:
Agreed, have updated in more recent commit.